website/docs/cloud-docs/workspaces/settings/health-assessments.mdx

```diff
@@ -0,0 +1,64 @@
+---
+page_title: Health Assessments - Workspaces - Terraform Cloud
+description: >-
+  Enable automatic jobs to check for lifecycle concerns, like whether your real-world infrastructure matches
+  your Terraform state file.
+tfc_only: true
+---
+
+# Health Assessments
+
+-> **Note:** Health Asessments (formerly Drift detection) is in beta and is available in the paid **Business** tier. Refer to [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) for details and  [provide feedback](https://forms.gle/BwjgvrKQcLbgqK3f7) about how we can improve this feature.
+
+The term health assessment is used to describe the regular, automatic inspection of a workspace to surface resource lifecycle concerns. Presently, this includes infrastructure drift detection but may include other concerns in the future.
+
+Infrastructure drift means that your real-world infrastructure no longer matches the configuration in your Terraform state file. Drift occurs when a user modifies resources outside of the Terraform workflow. For example, a colleague may update resource configuration directly in the cloud provider console during a production incident, changing the resource attributes from those in tracked in your state file.
+
+> **Hands On:** Try our [Manage Resource Drift](https://learn.hashicorp.com/tutorials/terraform/resource-drift) tutorial on HashiCorp Learn to review how to detect and resolve drift.
+
+Health assessments is a workspace-level setting. When enabled, Terraform Cloud will automatically run a health assessment for that workspace every 24 hours. The assessment checks the actual settings of your infrastructure against those tracked in your workspace's state file.
+
+## Requirements
+
+Workspaces require the following settings to receive health assessments:
+- Terraform version 0.15.4+
+- [Remote execution mode](/cloud-docs/workspaces/settings#execution-mode) or [Agent execution mode](/cloud-docs/agents#configuring-workspaces-to-use-the-agent) for Terraform runs
+
+## Enable Health Assessments
+
+To enable health assessments within a workspace:
+1. Verify that the workspace satisfies the [requirements](#requirements).
+1. Go to the workspace and click **Settings > General**.
+1. Select **Enable** under **Health Assessments**.
+1. Click **Save settings**.
+
+> **Note:** You may also enforce assessments across all workspaces in an organization from the [organization assessments settings](/cloud-docs/users-teams-organizations/organizations/#assessments)
```

Comment:
I'm thinking that this doesn't need to be a note --> it's a legitimate enablement method! I also wanted to add the tidbit here about how when you do that, it overrides whatever you did on the workspace because I think that's an important point!

```suggestion
You can also enforce health assessments for all eligible workspaces in an organization within the organization's **Assessments** settings. Enforcing health assessments at an organization-level overrides workspace-level settings. 
```